app/assets/javascripts/admin/addon/components/admin-plugins-list-item.gjs

```diff
@@ -30,9 +31,24 @@ export default class AdminPluginsListItem extends Component {
     }
   }
 
+  get isFiltered() {
+    if (!this.sidebarState.filter) {
+      return false;
+    }
+    return this.args.plugin.nameTitleized
+      .toLowerCase()
+      .match(this.sidebarState.filter);
+  }
+
   <template>
-    <tr data-plugin-name={{@plugin.name}}>
-      <td class="admin-plugins-list__row">
+    <tr
+      data-plugin-name={{@plugin.name}}
+      class={{concat
+        "admin-plugins-list__row"
+        (if this.isFiltered "-filtered")
```

Comment:
Sorry this is annoying, but I was thinking on my walk with the dogs this morning this should be a little more specific, maybe `-admin-search-filtered`